src/librustc_typeck/check/method/mod.rs

```diff
@@ -47,7 +47,7 @@ pub enum MethodError {
 
 // A pared down enum describing just the places from which a method
 // candidate can arise. Used for error reporting only.
-#[derive(Copy, Clone, PartialOrd, Ord, PartialEq, Eq)]
+#[derive(Debug, Copy, Clone, PartialOrd, Ord, PartialEq, Eq)]
```

Comment:
Nit: can you put Debug after Copy and Clone? In principle, I like to maintain alphabetical order, although I am not so great at it myself (e.g. I realize that PartialEq, Eq are not in order... bonus points if you fix that. ;)